src/attributes.jl

```diff
@@ -23,6 +23,13 @@ abstract type AbstractConstraintAttribute end
 
 const AnyAttribute = Union{AbstractInstanceAttribute, AbstractVariableAttribute, AbstractConstraintAttribute}
 
+"""
+    mustcopy(attr::Union{Type{AbstractInstanceAttribute}, Type{AbstractVariableAttribute}, Type{AbstractConstraintAttribute}})
+
+Returns whether the attribute affects the instance is thus mandatory to copy in `MOI.copy!` or whether it only affects how the instance is solved.
```

Comment:
"affects the instance" is unclear. Just say "is mandatory to copy" and give examples like `ObjectiveSense` compared with `GurobiLogLevel` (hypothetical).